assets/js/components/user-input/UserInputPreview.js

```diff
@@ -154,6 +155,14 @@ export default function UserInputPreview( props ) {
 						{ __( 'Review your answers', 'google-site-kit' ) }
 					</p>
 				) }
+				{ settingsdView && (
+					<p className="googlesitekit-settings-user-input__heading">
+						{ __(
+							'Edit your answers for more personalized metrics:',
+							'google-site-kit'
+						) }
+					</p>
+				) }
```

Comment:
I think we should preserve the previous layout: this block should go first wrapped with <Row><Cell>, then the div with `googlesitekit-user-input__preview` class.

```js
return (
    <Fragment>
        { settingsView && (
            <Row>
                <Cell>
                    <p ...>...</p>
                </Cell>
             </Row>
        ) }
        <div className={ classnames( 'googlesitekit-user-input__preview', { ... } ) }>
            ...
        </div>
    </Fragment>
)
```